plugins/modules/rds_instance.py

```diff
@@ -1403,6 +1427,12 @@ def validate_options(client, module: AnsibleAWSModule, instance: Dict[str, Any])
                 " source_db_instance_identifier"
             )
         )
+    if multi_tenant is not None and engine not in ["oracle-se2-cdb", "oracle-ee-cdb"]:
```

Comment:
```suggestion
    if multi_tenant and engine not in ("oracle-se2-cdb", "oracle-ee-cdb"):
```
module should fail only if the user is trying to define a `multi_tenant` database in a non-supported engine

Reply:
@abikouo The module will not fail, when multi_tenant is set to False and the engine is not one of the cdbs, with the suggestion you have made. Please correct me if I am wrong.